lib/messaging/handle.rb

```diff
@@ -104,8 +104,11 @@ def handler_logger
       end
 
       def handle_macro(message_class, &blk)
-        define_handler_method(message_class, &blk)
+        handler_method_name = define_handler_method(message_class, &blk)
+
         message_registry.register(message_class)
+
+        handler_method_name
       end
```

Comment:
To test this, call on a class and capture the return value, and then check the method name.